io/subsystem/src/main/java/org/wildfly/extension/io/OutboundBindAddressResourceDefinition.java

```diff
@@ -49,8 +49,7 @@
         .setRestartAllServices()
         .build();
 
-    static final SimpleAttributeDefinition BIND_ADDRESS = new SimpleAttributeDefinitionBuilder("bind-address", ModelType.STRING)
-        .setRequired(true)
+    static final SimpleAttributeDefinition BIND_ADDRESS = new SimpleAttributeDefinitionBuilder("bind-address", ModelType.STRING, true)
```

Comment:
I think the problem here is the default value.  A default value of numeric zero makes no sense for a string attribute whose value is meant to either be a resolvable hostname or some IP address notation.